airbyte-workers/src/main/java/io/airbyte/workers/temporal/sync/NormalizationActivity.java

```diff
@@ -12,14 +12,16 @@
 import io.airbyte.persistence.job.models.JobRunConfig;
 import io.temporal.activity.ActivityInterface;
 import io.temporal.activity.ActivityMethod;
+import java.util.UUID;
 
 @ActivityInterface
 public interface NormalizationActivity {
 
   @ActivityMethod
   NormalizationSummary normalize(JobRunConfig jobRunConfig,
                                  IntegrationLauncherConfig destinationLauncherConfig,
-                                 NormalizationInput input);
+                                 NormalizationInput input,
+                                 final UUID workspaceId);
```

Comment:
Can we add `workspaceId` to the `NormalizationInput` instead? This would line up better with how we are tracking labels for traces in the replication activity.

Reply:
Thanks, do we have a test for `NormalizationInput.workspaceId` being null? This will happen for the running syncs at the time of the deploy, if not handled properly, we'll have to manually clean up temporal errors.

Reply:
done